src/cosim/proxy/remote_slave.cpp

```diff
@@ -230,6 +230,21 @@ void cosim::proxy::remote_slave::release_state(state_index)
         "Proxyfmu does not support state saving");
 }
 
+cosim::serialization::node cosim::proxy::remote_slave::export_state(state_index) const
+{
+    throw error(
+        make_error_code(errc::unsupported_feature),
+        "Proxyfmu does not support state serialization");
+}
+
+cosim::slave::state_index cosim::proxy::remote_slave::import_state(
+    const cosim::serialization::node&)
+{
+    throw error(
+        make_error_code(errc::unsupported_feature),
+        "Proxyfmu does not support state serialization");
+}
+
```

Comment:
I guess we do not support this until #757 gets finished?

Reply:
That is something we should discuss, I guess. I just don't know the proxy-fmu code that well, and have enough on my plate with this, so I haven't prioritised it yet. That said, proxy-fmu is currently so important in our real-life use cases that I think it must be done at some point.

Reply:
We would need proxy-fmu support, but makes sense to leave that for last. Can be it's own PR following #757.